sdk/monitor/Azure.Monitor.Ingestion/src/Azure.Monitor.Ingestion.csproj

```diff
@@ -6,9 +6,10 @@
     <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
     <ApiCompatVersion>1.1.2</ApiCompatVersion>
     <PackageTags>Azure Monitor Ingestion</PackageTags>
-    <TargetFrameworks>$(RequiredTargetFrameworks);net461</TargetFrameworks>
+    <TargetFrameworks>$(RequiredTargetFrameworks);net462</TargetFrameworks>
```

Comment:
8.x packages weren't tested on net461 since it was deprecated. This leads to build errors if you try to depend on 8.x packages with a net461 target.